Feverup/decap-cms
# PNW-1326 - CMS fork migration

## 📱 Which kind of feature is this PR?

This PR includes all the work done through out the last year made on my personal fork of `netlify-cms`.

## ℹ️ What's this PR do?

You can see on the commits history everything that has been done but the main functionalities added are:
- Added `google auth` compatibility (This was necessary when we had multiple repositories)
- Added `media_library` validations like size check, name check, aspect-ratio check...
- Removed `backup` functionality to avoid issues with changes (Wasn't working well...)
- Added new `stack` changes workflow
- Added `prettier` linter

## 📱 How should this be tested?

> In order to test this make sure you are using `node v16`  

1. Checkout `feature/PNW-1326_cms-fork-migration`
2. Run `yarn install`

- [x] Test that `yarn build` works correctly
- [x] Test that the output generated on `netlify-cms/dist` works as the current one we have on `landings-cms`
- [ ] Test that the new workflow still works as expected checkout this [PR](https://github.com/Feverup/landings-cms/pull/357) to see the config necessary
